azerothcore/azerothcore-wotlk
# fix(Core/Guild): Implement gender in guild

## CHANGES PROPOSED:
-  Ported from https://github.com/TrinityCore/TrinityCore/commit/9f945d2f42
-  It is needed for locales like ruRU to show gender-specific names for classes in the guild roster window. (Thanks @Winfidonarleyan for testing the bug for me) 


## TESTS PERFORMED:
- Builds



## HOW TO TEST THE CHANGES:

1. Use a client that is supposed to change the name of classes depending on gender. i.e ruRU
2. Open the guild roster window and see if it changes the class name depending on your gender.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
